### PR TITLE
Generate device specific guides

### PR DESCRIPTION
**Pull request by iamsolankiamit**
Change-type: minor
Signed-off-by: Amit Solanki <amit@balena.io>

### CODE REVIEW

File: src/components/Guides.tsx [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+import tasksImg from '../img/tasks.png';
+import { DownloadEtcher } from './DownloadEtcher';
+import { LazyImage } from './LazyImage';
+import BalenaSdk from 'balena-sdk';
+
+const renderGenericInfo = (deviceName?: string) => (
+	<Txt.p>
+		Getting started on a <Txt.span bold>{deviceName}</Txt.span> is simple!
+		Follow these steps to download our ready-made operating system, flash it to
+		an SD Card, and begin crunching data to help scientists!
```
Comment (sradevski) [inline src/components/Guides.tsx:16]: This is not always an sd card, so maybe we want to say either `SD card or storage drive`, but others might have a better suggestion :)

**Review by sradevski — APPROVED** [review]
I think we can merge this and do https://github.com/balenalabs/fold-for-covid-landing/issues/64 in a separate PR, but that needs to be handled as well.

File: src/components/Guides.tsx (hunk 1 above)
Comment (iamsolankiamit) [reply src/components/Guides.tsx:16]: @sradevski Yes, this was done to enable that. We'll need a list of devices without SD card to do that.

**Comment by iamsolankiamit** [comment]
@balena-ci rebase

**Pull request merged**